inlong-audit/audit-proxy/src/main/java/org/apache/inlong/audit/source/ServerMessageHandler.java

```diff
@@ -147,6 +151,13 @@ private AuditReply handleRequest(AuditRequest auditRequest) throws Exception {
         List<AuditMessageBody> bodyList = auditRequest.getMsgBodyList();
         int errorMsgBody = 0;
         for (AuditMessageBody auditMessageBody : bodyList) {
+            long msgDays = messageDays(auditMessageBody.getLogTs());
+            if (msgDays >= this.msgValidThresholdDays) {
+                String msg = String.format("Discard the data as it is from" + msgDays +
```

Comment:
Unused `String.format`, because not use `%s` in string here.

BTW, please add one blank after `is from` and before `days ago`.